eternalcore/src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -323,20 +320,20 @@ public void onEnable() {
 
             // Arguments (include optional)
             .argument(String.class, "player",   new PlayerNameArg(server))
-            .argument(GameMode.class,               new GameModeArgument(this.userProvider, this.languageManager))
-            .argument(NoticeType.class,             new NoticeTypeArgument(this.userProvider, this.languageManager))
-            .argument(Warp.class,                   new WarpArgument(this.warpManager, this.languageManager, this.userProvider))
+            .argument(GameMode.class,               new GameModeArgument(viewerProvider, this.languageManager))
+            .argument(NoticeType.class,             new NoticeTypeArgument(this.viewerProvider, this.languageManager))
+            .argument(Warp.class,                   new WarpArgument(this.warpManager, this.languageManager, this.viewerProvider))
             .argument(Enchantment.class,            new EnchantmentArgument(this.viewerProvider, this.languageManager))
             .argument(World.class,                  new WorldArgument(server))
             .argument(User.class,                   new UserArgument(this.viewerProvider, this.languageManager, server, this.userManager))
             .argument(Player.class,                 new PlayerArgument(this.viewerProvider, this.languageManager, server))
-            .argument(Player.class, "request",  new RequesterArgument(this.teleportRequestService, this.languageManager, this.userProvider, server))
+            .argument(Player.class, "request",  new RequesterArgument(this.teleportRequestService, this.languageManager, this.viewerProvider, server))
```

Comment:
nie wiem czy to wina githuba, ale te spacje można wyrównać

Reply:
Nie to nie wina githuba, a bardziej intelki 😄